profiles/killbill/src/test/java/org/killbill/billing/jaxrs/TestEntitlement.java

```diff
@@ -299,6 +300,36 @@ public void testCreateEntitlementInTheFuture() throws Exception {
         Assert.assertTrue(objFromJson.equals(entitlementJson));
     }
 
+    @Test(groups = "slow", description = "Can create an entitlement with an account with autoPayOff")
+    public void testCreateEntitlementWithAutoPayOff() throws Exception {
+        final Account accountJson = createAccount();
+        assertNotNull(accountJson);
+
+        // assign autoPaymentOff tag to account
+        Tags tags = killBillClient.createAccountTag(accountJson.getAccountId(), new UUID(0L, 1L), requestOptions);
+        assertEquals(tags.get(0).getTagDefinitionName(), "AUTO_PAY_OFF");
+
+        // verify that number of invoices and payments for account is still 0
+        assertEquals(killBillClient.getInvoicesForAccount(accountJson.getAccountId(), requestOptions).size(), 0);
+        assertEquals(killBillClient.getPaymentsForAccount(accountJson.getAccountId(), requestOptions).size(), 0);
```

Comment:
Scratch that comment... got confused with your comment. This is unrelated to the tag. All good.